### DISCUSSION TITLE
any example video for deploying flet app or to convert flet app into flutter app

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#564] [Q&A]
any example video for deploying flet app or to convert flet app into flutter app because can't find any video or in documentation proper steps to deploy an flet app

**Comment by ndonkoHenri** [comment]
Do you mean this? https://flet.dev/docs/guides/python/deploying-web-app

**Reply by plugeit** [reply]
I need a video or a proper documentation the link which you have provided can't solve my problem please help. Any video which gives step by step guide to deploy it or you have deployed it or anyone.
